package.json

```diff
@@ -3,19 +3,26 @@
   "version": "6.0.0",
   "description": "a collection of utility functions for Ethereum",
   "main": "dist/index.js",
+  "types": "./dist/index.d.ts",
   "files": [
     "dist"
   ],
   "scripts": {
-    "coverage": "npm run build:dist && istanbul cover _mocha",
+    "build": "tsc -p ./tsconfig.prod.json",
+    "prepublishOnly": "npm run test && npm run build",
+    "coverage": "npm run build && istanbul cover _mocha",
     "coveralls": "npm run coverage && coveralls <coverage/lcov.info",
-    "lint": "standard",
-    "prepublishOnly": "npm run test && npm run build:dist",
+    "docs:build": "typedoc --out docs --mode file --readme none --theme markdown --mdEngine github src",
+    "format": "prettier --list-different '**/*.{ts,json,md}'",
+    "format:fix": "prettier --write '**/*.{ts,json,md}'",
+    "lint": "npm run format && npm run tslint && npm run tsc",
+    "lint:fix": "npm run format:fix && npm run tslint:fix && npm run tsc",
     "test": "npm run lint && npm run test:node && npm run test:browser",
-    "test:browser": "npm run build:dist && karma start karma.conf.js",
-    "test:node": "npm run build:dist && istanbul test mocha -- --reporter spec",
-    "build:dist": "babel index.js --source-root ./ -d ./dist",
-    "build:docs": "documentation build ./index.js --github --sort-order='alpha' -f md > ./docs/index.md"
+    "test:browser": "npm run build && karma start karma.conf.js",
+    "test:node": "npm run build && istanbul test mocha -- --reporter spec",
+    "tsc": "tsc --noEmit",
+    "tslint": "tslint -p ./tsconfig.json -e node_modules/**/* -e **/node_modules/**/* -e dist/**/* **/*.ts",
+    "tslint:fix": "tslint --fix --format stylish -p ./tsconfig.json -e node_modules/**/* -e **/node_modules/**/* -e dist/**/* **/*.ts"
```

Comment:
Can we already use the new unified script command here and for the different commands and update the associated `ethereumjs-config-*` dev dependencies?